sky/spot/spot_utils.py

```diff
@@ -351,12 +355,12 @@ def dump_spot_job_queue() -> str:
         if end_at is None:
             end_at = time.time()
 
-        job_start_at = job['last_recovered_at'] - job['job_duration']
+        job_launched_at = job['last_recovered_at'] - job['job_duration']
```

Comment:
Seems like these are the only "launched at" left in the spot folder. Is it expected?

Reply:
Good catch! Fixed. Thanks!

Reply:
Good catch! Fixed. Thanks!